lab_2_tokenize_by_bpe/main.py

```diff
@@ -36,7 +59,16 @@ def count_tokens_pairs(
     :param word_frequencies: dictionary in the form of <preprocessed word: number of occurrences>
     :return: dictionary in the form of <token pair: number of occurrences>
     """
-
+    if not isinstance(word_frequencies, dict):
+        return None
+    pairs_dictionary = {}
+    for token in word_frequencies:
+        for i, index in enumerate(token):
```

Comment:
Isn't the order reversed? First is the index, the second is the element? BTW, `i` is not a good variable name.

Reply:
Fixed